### PR TITLE
Call control

### PR DESCRIPTION
**Pull request by rajjy1**

### CODE REVIEW

File: src/Telnyx.Example/Program.cs [hunk 1]
```diff
@@ -4,12 +4,15 @@ namespace Telnyx.Example
...
+
         static void Main(string[] args)
         {
             Console.WriteLine("Starting Examples...");
 
-            TelnyxConfiguration.SetApiBase($"https://api.telnyx.com/v2");
-            TelnyxConfiguration.SetApiKey("YOUR_API_KEY");
+            TelnyxConfiguration.SetApiBase("https://api.telnyx.com/v2");// ($"http://localhost:12111/v2");
+            TelnyxConfiguration.SetFilesBase("http://api.telnyx.com/v2"); //($"http://localhost:12111/v2");
+            TelnyxConfiguration.SetApiKey("KEY0170CBFE0003DDA859D411D1013E1F90_e4Hp11KrxKgfFlqUxLgg9H");
```
Comment (thedonmon) [inline src/Telnyx.Example/Program.cs:15]: ApiKey left in here

Comment (rajjy1) [reply src/Telnyx.Example/Program.cs:15]: Removing it.

File: src/Telnyx.net/Infrastructure/Urls.cs [hunk 2]
```diff
@@ -12,11 +12,31 @@ internal static class Urls
...
+
+        /// <summary>
+        /// Gets defaultBaseFilesUrl
+        /// </summary>
+        internal static string DefaultBaseFilesUrl => "https://files.Telnyx.com/v1";
+
+        /// <summary>
+        /// Gets baseFilesUrl
+        /// </summary>
+        internal static string BaseFilesUrl => TelnyxConfiguration.GetFilesBase();
```
Comment (eoghantelnyx) [inline src/Telnyx.net/Infrastructure/Urls.cs:40]: @rajjy1 we can remove these

File: src/Telnyx.net/LICENSE [hunk 3]
```diff
@@ -1,7 +1,6 @@
 The MIT License
 
-Copyright (c) 2019- Telnyx (https://telnyx.com)
-Copyright (c) 2010-2019 Stripe (https://stripe.com)
+Copyright (c) 2019 Telnyx (https://telnyx.com)
```
Comment (eoghantelnyx) [inline src/Telnyx.net/LICENSE:3]: Please revert this to include the Stripe line @rajjy1

File: src/TelnyxTests/Infrastructure/RequestorTest.cs [hunk 4]
```diff
@@ -17,7 +17,7 @@ public void SetsHeaders()
         {
             RequestOptions options = new RequestOptions
             {
-                ApiKey = "TESTKEY",
+                ApiKey = "sk_key",
```
Comment (eoghantelnyx) [inline src/TelnyxTests/Infrastructure/RequestorTest.cs:20]: Please change this to `TESTKEY`

**Comment by thedonmon** [comment]
Might need to update this branch with a merge from master @rajjy1

File: src/TelnyxTests/Infrastructure/RequestorTest.cs (hunk 4 above)
Comment (rajjy1) [reply src/TelnyxTests/Infrastructure/RequestorTest.cs:20]: Done

File: src/Telnyx.net/LICENSE (hunk 3 above)
Comment (rajjy1) [reply src/Telnyx.net/LICENSE:3]: Done

**Comment by rajjy1** [comment]
> branch

Done.

**Pull request merged**